dash/dash-renderer/src/actions/paths.js

```diff
@@ -76,7 +76,7 @@ export function getPath(paths, id) {
             return false;
         }
         const values = props(keys, id);
-        const pathObj = find(propEq('values', values), keyPaths);
+        const pathObj = find(propEq(values, 'values'), keyPaths);
```

Comment:
Ah it was inverted, strange because it feel more natural with the prop name first.